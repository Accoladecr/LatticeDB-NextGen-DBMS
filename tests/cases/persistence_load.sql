LOAD DATABASE 'tmp.ldb';
SELECT id, v FROM p;
EXIT;
