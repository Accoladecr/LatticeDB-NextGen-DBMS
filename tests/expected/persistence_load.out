Loaded tmp.ldb
id | v
'x' | 42
Bye.
